ThePensionsRegulator.Frontend/HtmlGeneration/ComponentGenerator.TprSearch.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.AspNetCore.Mvc.Rendering;
+
+namespace ThePensionsRegulator.Frontend.HtmlGeneration
+{
+    public partial class ComponentGenerator
+    {
+        internal const string SearchElement = "aside";
+
+        public virtual TagBuilder GenerateTprSearch(string faqApiUrl)
+        {
+            var section = new TagBuilder(SearchElement);
+            section.AddCssClass("tpr-search");
+            section.Attributes.Add("id", "tpr-search");
```

Comment:
So long as there's only ever one of these on a page this works, but two would make a hard-coded id invalid.